lib/shared/addon/components/pretty-json/template.hbs

```diff
@@ -1,5 +1,5 @@
 {{#if json}}
-  <pre><code>{{json}}</code></pre>
+  <pre style="background: #fff;"><code style="color: #000;">{{json}}</code></pre>
```

Comment:
Can you turn this into a class and verify that it looks good when dark mode is turned on as well?